data/countries_grammar.csv

```diff
@@ -40,6 +40,7 @@ CIV,,,,,,,,,,,
 CMR,,,,,,,,,,,
 COD,,1,1,,,1,,,,,
 COG,,1,1,,,1,,,,,
+COK,1,1,,,,,,,,,,
```

Comment:
@abigailmarshall: It looks like this row now has 12 commas, indicating 13 columns. However it should only have a total of 11 commas, delimiting the 12 columns. No big deal at all, but I wonder if you had made these edits online via Github.com or using a desktop program like MS Excel?